cloud/mutagen/wrapper.go

```diff
@@ -123,6 +125,14 @@ func execMutagen(args []string) error {
 	return nil
 }
 
+func env(envVars map[string]string) []string {
+	customEnv := []string{}
```

Comment:
You can start out with the environment to avoid the extra append.

```suggestion
	customEnv := os.Environ()
```